src/Utility/Rsync.php

```diff
@@ -43,7 +43,11 @@ public function call($hostname, $source, $destination, array $config = [])
 
         $this->pop->command($hostname, $rsync);
 
-        $process = new Process($rsync);
+        if (method_exists('Symfony\Component\Process\Process', 'fromShellCommandline')) {
```

Comment:
Maybe replace with `if (method_exists(Process::class, 'fromShellCommandline'))`?

Reply:
I agree that would be slightly cleaner. For now I used the exact same fix that was already present in e.g. `Deployer\Utility\ProcessRunner` for consistency. The best long-term solution would probably be to have a utility function somewhere in the deployer codebase to contain this compatibility layer.